File: antia_numerical_methods@Beliavsky/f77/sub8/fbeta.f
!	To calculate the integrand for incomplete beta function
!       It is used by function BETAI
!
!	A,B : Arguments for the complete Beta function passed through
!               a common block
!	X : (input) Upper limit of integration defining the incomplete
!               Beta function
!
!	Required routines : none

      FUNCTION FBETA(X)
      IMPLICIT REAL*8(A-H,O-Z)
      COMMON/BETA/A,B
      FBETA=X**(A-1)*(1-X)**(B-1)
      END

File: projects/antia_numerical_methods@Beliavsky/c/fun/fbeta.c
/*	To calculate the integrand for incomplete beta function
	The variable AA, BB need to be defined outside

	Required routines : none
*/

double AA, BB;

#include <math.h>

	double fbeta(double x)
{
	return pow(x,AA-1)*pow(1-x,BB-1);
}

